mobile/src/main/res/values/strings.xml

```diff
@@ -950,6 +950,8 @@ Would you like to share analytics with the Tella team?</string>
   <string name="Drafts_Header_Message">These are reports you are working on.</string>
   <string name="Outboxes_Header_Message">These reports are ready to be submitted</string>
   <string name="Submitted_Header_Message">These are reports that you have successfully submitted.</string>
-  <string name="Connections_description">Learn more about connections here. If you\n work with an organization, you can ask them\n about which connection to use.</string>
+  <string name="Connections_description">Learn more about connections here. If you
```

Comment:
@ahlem-jarrar @wafalakhal  do you think it's a good practice to relay on the translators to keep the enter here ? if they remove it , the string will show fine as well?